roles/openshift_storage_glusterfs/tasks/glusterfs_common.yml

```diff
@@ -313,7 +313,10 @@
   - glusterfs_storageclass or glusterfs_s3_deploy
 
 - include_tasks: glusterblock_deploy.yml
-  when: glusterfs_block_deploy
+  when: 
```

Comment:
Tox doesn't like trailing space here

Reply:
@mjudeikis Always run tox before submitting a PR. :) I say this as someone who often forgets to do that.

Reply:
dam.  But are we fine now to just ignore atomic host? (i think we dont have other options :/ )

Reply:
It's what we've been doing, unfortunately. I need to sit down with my team and figure out what the strategy on Atomic is, I just haven't had the time.

Reply:
sorted... sorry, too many contexts switches these days...